client/lib/application/src/application-lifecycle/application-lifecycle.jsx

```diff
@@ -1,25 +1,124 @@
 import React from 'react';
-/*eslint-disable */
+import PropsExposer from 'common/src/components/pure/PropsExposer.jsx';
+import ComboBox from 'common/src/components/pure/ComboBox.jsx';
+import ThreeColumns from 'common/src/components/pure/ThreeColumns.jsx';
```

Comment:
I'd really like to introduce the components folder in common/src/ as there is already a data folder. in long term I figured to move all components that are on top level to a components sub folder.